themes/src/main/resources-community/theme/base/login/messages/messages_es.properties

```diff
@@ -104,48 +152,74 @@ role_manage-clients=Gestionar clientes
 missingEmailMessage=Por favor indica tu email.
 missingUsernameMessage=Por favor indica tu usuario.
 missingPasswordMessage=Por favor indica tu contrase\u00F1a.
 missingTotpMessage=Por favor indica tu c\u00F3digo de autenticaci\u00F3n
+missingTotpDeviceNameMessage=Por favor especifique el nombre del dispositivo.
 notMatchPasswordMessage=Las contrase\u00F1as no coinciden.
 
 invalidPasswordExistingMessage=La contrase\u00F1a actual no es correcta.
+invalidPasswordBlacklistedMessage=Contrase\u00F1a no v\u00E1lida: la contrase\u00F1a est\u00E1 en la lista negra.
 invalidPasswordConfirmMessage=La confirmaci\u00F3n de contrase\u00F1a no coincide.
 invalidTotpMessage=El c\u00F3digo de autenticaci\u00F3n no es v\u00E1lido.
 
 usernameExistsMessage=El nombre de usuario ya existe
 emailExistsMessage=El email ya existe
 
-federatedIdentityEmailExistsMessage=Ya existe un usuario con este email. Por favor accede a la gesti\u00F3n de tu cuenta para enlazarlo.
-federatedIdentityUsernameExistsMessage=Ya existe un usuario con este nombre de usuario. Por favor accede a la gesti\u00F3n de tu cuenta para enlazarlo.
+federatedIdentityExistsMessage=El usuario con {0} {1} ya existe. Por favor inicie sesi\u00F3n en la administraci\u00F3n de la cuenta para vincular la cuenta.
+
+confirmLinkIdpTitle=La cuenta ya existe
+federatedIdentityConfirmLinkMessage=El usuario con{0} {1} ya existe. \u00BFC\u00F3mo desea continuar?
+federatedIdentityConfirmReauthenticateMessage=Autent\u00EDquese para vincular su cuenta con {0}
+nestedFirstBrokerFlowMessage=El {0} usuario {1} no est\u00E1 vinculado con ningun usuario conocido.
+confirmLinkIdpReviewProfile=Revisar perfil
+confirmLinkIdpContinue=Agregar a una cuenta existente
 
 configureTotpMessage=Tienes que configurar la aplicaci\u00F3n m\u00F3vil de identificaci\u00F3n para activar tu cuenta.
 updateProfileMessage=Tienes que actualizar tu perfil de usuario para activar tu cuenta.
 updatePasswordMessage=Tienes que cambiar tu contrase\u00F1a para activar tu cuenta.
+resetPasswordMessage=Usted necesita cambiar su contrase\u00F1a
 verifyEmailMessage=Tienes que verificar tu email para activar tu cuenta.
+linkIdpMessage=Usted necesita verificar su direcci\u00F3n de correo electr\u00F3nico para vincular su cuenta con {0}.
 
 emailSentMessage=En breve deber\u00EDas recibir un mensaje con m\u00E1s instrucciones
 emailSendErrorMessage=Fall\u00F3 el env\u00EDo del email, por favor int\u00E9ntalo de nuevo m\u00E1s tarde.
 
 accountUpdatedMessage=Tu cuenta se ha actualizado.
 accountPasswordUpdatedMessage=Tu contrase\u00F1a se ha actualizado.
 
+delegationCompleteHeader=Inicio de sesi\u00F3n exitoso
+delegationCompleteMessage=Usted puede cerrar esta ventana del navegador y volver a la aplicaci\u00F3n de consola.
+delegationFailedHeader=Error en el inicio de sesi\u00F3n
+delegationFailedMessage=Usted puede cerrar esta ventana del navegador y volver a la aplicaci\u00F3n de consola, e intantar iniciar sesi\u00F3n de nuevo.
```

Comment:
Remove "Usted"
"Puede cerrar..."

Reply:
Also, typo; "intantar" should be "intentar"